### ISSUE TITLE
v0.9.0 - New features & Bug fixes

### ISSUE THREAD

**Issue by Monte9** [react-native-community/react-native-elements#125]
We are working on our next release v0.9.0 just in time for the New Year. Here's how to get your hands on the next release before it's out:

Replace this line in your `package.json`:
```
"react-native-elements": "git://github.com/react-native-community/react-native-elements.git#v0.9.0"
```
Then
```
npm upgrade
```
Enjoy! 🎉

# v0.9.0 

### Features
- ListItem subtitle prop can now render custom Views 
- SideMenu now recognizes gestures to open & close it (switched to using react-native-side-menu)
- Added Grid Component 
- Added Slider Component

### Bug fixes
- fixed ListItem marginLeft issue; it's now flush on both ends

> this list will be updated as more features/issues go into the release

**Comment by ngduc** [comment]
@Monte9 Happy New Year 2017! Thanks for this great set of elements!
Can't wait to see 0.9.0 released :)

**Comment by Monte9** [comment]
Thanks @ngduc! Credit goes to @dabit3 for putting this together!

**Comment by dabit3** [comment]
Thanks all, honestly the credit definitely goes to @Monte9 for doing almost all of the work for this release, as well as all of those who contributed in the past couple of weeks. Thanks so much!

**Comment by Monte9** [comment]
Released v0.9.0. 🎉 

Feel free to file an issue if you come across any bugs/issues with this release. Closing this issue now.
